.github/workflows/release.yaml

```diff
@@ -15,9 +15,9 @@ env:
     RELATED_IMAGE_CSIV1_EXTERNAL_ATTACHER_V3=quay.io/k8scsi/csi-attacher:v3.1.0
     RELATED_IMAGE_CSIV1_EXTERNAL_RESIZER=quay.io/k8scsi/csi-resizer:v1.1.0
     RELATED_IMAGE_STORAGEOS_INIT=storageos/init:v2.1.2
-    RELATED_IMAGE_STORAGEOS_NODE=soegarots/node:v2-release-v2.7.0-24
+    RELATED_IMAGE_STORAGEOS_NODE=storageos/node:v2-release-v2.7.0-24
```

Comment:
This node image points to a pre build on soegarots, so this change leads to and image pull issue.